### DISCUSSION TITLE
live streaming feature for product seling. ( Live Streaming Layout / Section )

### DISCUSSION THREAD

**Discussion by ecomdeveloperbd** [inspireui/support#5629] [New Feature/Ideas Requests]
A very useful product-selling medium these days is the live streaming feature. This feature is currently not included in existing WooCommerce apps. If this feature can be added before Black Friday, hopefully, these apps will sell best this Black Friday. You can highlight this feature on Black Friday. This option is requested to be added in the next version

https://prnt.sc/1IpBIl_4qV1Q

**Comment by ecomdeveloperbd** [comment]
add this plugin or if your better than plugin used...
best suggest if any process include Facebook live option add this in app.. it's better case fb live unlimited audience same time view


but this plugin audience limited 

https://wordpress.org/plugins/live-shopping-video-streams/

**Comment by usamaahmed5656** [comment]
from manager app the seller has a option to go live when he hit go live ,when any of the seller is live then on fluxstore multivendor app category screen(store list screen) the live text should be shown with the store name ..there should also be a widget of showing live ongoing streams on front page of the customers app and by mean showing streaming on customer's app homepage i am not wanting to show the video streaming because it can increase the loadtime of homepage but a widget that can filter from store list of which sellers are live rightnow and show the name of the store with banner image with live text on it

the third and most important thing is when a seller is live streaming and customer is watching them on their fluxstore multivendor screeen ...soo the streaming screen should have a side bar which is scrollable and is showing all on sale products or may be all products of that particular store ,, this feature will help customers to buy easily the things the seller is showing on live streaming

1 ) sellers going live from manager app
2 ) customers are able to see which sellers are live on fluxstore multivendor
3 ) sidebar scrollable products screen on streaming screen

hope you understand the idea and like it becasue this will get fluxstore to next level

**Comment by ecomdeveloperbd** [comment]
Dear Sir, For your convenience I have provided a nice video on how Facebook Live feature videos are added in this commerce apps in the below link.  Hope if you can add this feature your app will be very nice app.  This feature includes live videos, previous videos, and all videos together.  I did not describe much, hope you will understand after watching the video.  For further understanding, I have given the link to the app.  Hope to add such feature soon.  Live product selling is very popular nowadays.  I think it would be best to add facebook live video without using any plugin.  Thank you 


https://streamable.com/mjq89q

https://play.google.com/store/apps/details?id=com.ajkerdeal.app.android

Story videos are shown within apps like TikTok Stories. using username. In the same way, live videos of a single user's Facebook page and previous videos can be shown using the Facebook username. On the other hand, in the case of multi-vendor websites or apps, there are multiple users. Username can be set. Multiple users can be set



।।।।।।।।

Another thing you can try and see if the reel videos can be shown using the Facebook page or profile username. And of course, the link can be used in the case of the videos, thanks I may have many English sentence and grammatical mistakes, please forgive me.
